core/src/main/java/io/questdb/cairo/vm/MemoryCMRImpl.java

```diff
@@ -72,6 +72,12 @@ public void extend(long newSize) {
     public void of(FilesFacade ff, LPSZ name, long extendSegmentSize, long size, int memoryTag) {
         this.memoryTag = memoryTag;
         openFile(ff, name);
+        if (size < 0) {
+            size = ff.length(fd);
+            if (size < 0) {
+                throw CairoException.instance(ff.errno()).put("File not found: ").put(name);
```

Comment:
may be reword the exception message to be factual, e.g. "could not get length [name="

Reply:
done in 95dab2b